opencollective.svg

```diff
@@ -0,0 +1 @@
h.987c.04 0 .06-.02.06-.06v-.987c0-.04-.02-.06-.06-.06zm-2.035 1.892h-1.107c-.02 0-.04-.02-.04-.04v-2.517c0-.04-.02-.06-.06-.06H98.6c-.04 0-.06.02-.06.06V27c0 .02-.02.04-.04.04h-1.087c-.04 0-.06.02-.06.06v.987c0 .04.02.06.06.06H98.5c.02 0 .04.02.04.04v4.7c0 .04.02.06.06.06h.987c.04 0 .06-.02.06-.06v-4.7c0-.02.02-.04.04-.04h1.107c.04 0 .06-.02.06-.06V27.1c0-.02-.04-.06-.06-.06z"/><use xlink:href="#B"/><path d="M87.738 31.943a1.92 1.92 0 0 1-1.45-.644c-.02-.04-.02-.08.02-.1l3.383-1.43.987-.423c.04-.02.04-.04.04-.06a3.04 3.04 0 0 0-3.08-2.356c-1.57.06-2.88 1.33-2.94 2.9a3.05 3.05 0 0 0 3.06 3.2c1.268 0 2.356-.785 2.82-1.892.02-.04 0-.08-.04-.08l-.987-.18c-.02 0-.06 0-.06.04-.362.604-1.027 1.027-1.752 1.027zm0-3.906c.584 0 1.107.262 1.45.664.02.04.02.08-.02.1l-3.302 1.4a.06.06 0 0 1-.081-.04V30c-.02-1.107.866-1.973 1.953-1.973zm-3.886-3.624h-.987c-.04 0-.06.02-.06.06v8.416c0 .04.02.06.06.06h.987c.04 0 .06-.02.06-.06v-8.416c0-.04-.04-.06-.06-.06zm-2.155 0H80.7c-.04 0-.06.02-.06.06v8.416c0 .04.02.06.06.06h.987c.04 0 .06-.02.06-.06v-8.416c-.02-.04-.04-.06-.06-.06zm-4.832 3.624c1.087 0 1.953.886 1.953 1.953a1.97 1.97 0 0 1-1.953 1.953 1.97 1.97 0 0 1-1.953-1.953c0-1.067.866-1.953 1.953-1.953zm0-1.107a3.06 3.06 0 1 0 0 6.12 3.06 3.06 0 1 0 0-6.12z"/><use xlink:href="#B" x="-23.075"/><path d="M61.24 26.93a2.33 2.33 0 0 0-1.732.886c-.02.04-.1.02-.1-.02v-.705c0-.02-.02-.06-.06-.06h-.987c-.02 0-.06.02-.06.06v5.818c0 .02.02.06.06.06h.987c.02 0 .06-.02.06-.06V30c0-1.107.745-1.892 1.812-1.892 1.4 0 1.832.926 1.832 2.013v2.78c0 .02.02.06.06.06h.987c.02 0 .06-.02.06-.06v-2.77c.02-1.7-.785-3.2-2.92-3.2zm-6.503 5.013a1.92 1.92 0 0 1-1.45-.644c-.02-.04-.02-.08.02-.1l3.383-1.43.987-.423c.04-.02.04-.04.04-.06a3.04 3.04 0 0 0-3.08-2.356c-1.57.06-2.88 1.33-2.94 2.9a3.05 3.05 0 0 0 3.06 3.2c1.268 0 2.356-.785 2.82-1.892.02-.04 0-.08-.04-.08l-.987-.18c-.02 0-.06 0-.06.04-.362.604-1.007 1.027-1.752 1.027zm0-3.906c.584 0 1.107.262 1.45.664.02.04.02.08-.02.1l-3.302 1.4a.06.06 0 0 1-.081-.04V30a1.96 1.96 0 0 1 1.953-1.973zm-6.685-1.167c-.765.02-1.37.322-1.792.825-.04.04-.1.02-.1-.04v-.564c0-.04-.02-.06-.06-.06h-.987c-.04 0-.06.02-.06.06v8.436c0 .04.02.06.06.06h.966c.04 0 .06-.02.06-.06v-3.322c0-.06.06-.08.1-.04.423.503 1.047.825 1.852.825 1.873 0 3.362-1.7 3-3.624-.242-1.47-1.57-2.517-3.04-2.496zm.383 4.993a1.96 1.96 0 0 1-2.255-2.255c.14-.805.785-1.47 1.6-1.6a1.96 1.96 0 0 1 2.255 2.255c-.12.805-.785 1.45-1.6 1.6zm-7.007-4.933a3.06 3.06 0 1 0 0 6.12 3.06 3.06 0 1 0 0-6.12zm0 1.107c1.087 0 1.953.886 1.953 1.953a1.97 1.97 0 0 1-1.953 1.953 1.97 1.97 0 0 1-1.953-1.953 1.97 1.97 0 0 1 1.953-1.953z"/></g></g><defs ><path id="B" d="M94.322 31.943a1.97 1.97 0 0 1-1.933-2.335c.12-.685.805-1.39 1.5-1.53a1.95 1.95 0 0 1 1.893.604c.02.02.06.02.08 0l.705-.705c.02-.02.02-.06 0-.08-.584-.644-1.45-1.027-2.396-.987-1.55.08-2.84 1.35-2.92 2.9-.08 1.752 1.31 3.2 3.06 3.2.886 0 1.69-.383 2.255-.987.02-.02.02-.04 0-.06l-.705-.705c-.02-.02-.06-.02-.08 0-.342.443-.866.685-1.45.685z"/></defs></svg>
```

Comment:
Can we host these icons on some image hosting website instead? 🤔 
Just trying to avoid packaging files in project which doesn't really add any functionality to the project itself.

Reply:
Yes, but we would rely on them.